buzsaki_lab_to_nwb/huszar_hippocampus_dynamics/behaviorinterface.py

```diff
@@ -122,8 +170,18 @@ def run_conversion(self, nwbfile: NWBFile, metadata: dict, stub_test: bool = Fal
 
         unit = "cm"
         conversion = 100.0  # cm to m TODO: Double check if this is the meaning.
-        reference_frame = "TBD"
+        reference_frame = "Arbitrary, camera"
+
+        nest_depth = len(mat_file["behavior"]["trials"]["position_trcat"])
+
+        # Merge unique descriptions if there are nested entries inside the behavior file
         description = mat_file["behavior"]["description"]
+
+        if nest_depth > 1:
+            description = ";".join(
+                np.unique(mat_file["behavior"]["description"])
+            )  # NOTE: Description is an array in this case
+
         pos_obj = Position(name=description)
         spatial_series_object = SpatialSeries(
             name="position",
```

Comment:
Should be `SpatialSeries{descriptor that makes this series different from the others}`

Reply:
Likewise the `linearized_position` should be `SpatialSeriesLinearized` or `LinearizedSpatialSeries` for similarity to https://github.com/catalystneuro/buzsaki-lab-to-nwb/blob/bd7b4604443c37d80590e81c2883d83adbd70895/buzsaki_lab_to_nwb/petersen/petersenmiscdatainterface.py#L128